deploy/observability/dynamo_local_resource_monitor.py

```
@@ -0,0 +1,582 @@
+def _main_worker(conn, gpu_count, interval_sec):
+    """Subprocess: CPU (overall + per-name groups) + GPU mem/util + network at 5/s; temperature at 1/s."""
+    import psutil as _ps
+
+    handles = []
+    if gpu_count > 0:
+        try:
+            import pynvml as _nvml
+
+            _nvml.nvmlInit()
+            handles = [_nvml.nvmlDeviceGetHandleByIndex(i) for i in range(gpu_count)]
+        except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)